src/main.c

```diff
@@ -524,9 +524,10 @@ static void process_command_line_arguments(char *argv[], char **options)
 		fclose(fp);
 	}
 #endif
-
-	/* read all configurations from a config file */
-	(void)read_config_file(g_config_file, options);
+	if (cmd_line_opts_start == 2) {
```

Comment:
In the documentation it is said that we can use commande line's options OR the config file. But in fact the config file have to be created (at least empty) to be able the launch the server with options. 
With this modification, if the command line is defined with options (Cf. test L.495 - /* Should we use a config file ? */) it doesn't try to read a config file. 
I didn't notice sides effects with this modification according to want is described in the doc and the way it is coded.